### PR TITLE
Add `--data-from-json` option for initialization of emulator data

### PR DESCRIPTION
**Pull request by ohaibbq**
The JSON parser is exponentially faster than the YAML parser for large files

### CODE REVIEW

File: server/testdata/data.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                  "interval": "1-6 15 0:0:0"
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
```
Comment (ethan-oro) [inline server/testdata/data.json:108]: nit: newline

File: cmd/bigquery-emulator/main.go [hunk 2]
```diff
@@ -23,6 +23,7 @@ type option struct {
 	LogFormat    server.LogFormat `description:"specify the log format (console/json)" long:"log-format" default:"console"`
 	Database     string           `description:"specify the database file if required. if not specified, it will be on memory" long:"database"`
 	DataFromYAML string           `description:"specify the path to the YAML file that contains the initial data" long:"data-from-yaml"`
+	DataFromJSON string           `description:"specify the path to the JSON file that contains the initial data (faster for large files)" long:"data-from-json"`
```
Comment (ethan-oro) [inline cmd/bigquery-emulator/main.go]: do you have a sense at what size, in general, you would recommend switching from yaml to json? could be helpful to have a general sense (either here or in a doc somewhere) to guide folks

**Review by ethan-oro — APPROVED** [review]
looks amazing ✨

**Review by ageiduschek — APPROVED** [review]
No comments outside of Ethan's!

File: cmd/bigquery-emulator/main.go (hunk 2 above)
Comment (ohaibbq) [reply cmd/bigquery-emulator/main.go]: updated the helpdoc to say "multi-megabyte"

**Pull request merged**